src/symbols.rs

```diff
@@ -21,6 +21,12 @@ pub mod block {
         pub empty: &'static str,
     }
 
+    impl Default for Set {
+        fn default() -> Self {
+            THREE_LEVELS
```

Comment:
NINE_LEVELS is probably more generally useful for these as it shows more information to the user whereas three levels throws some away.

Reply:
OK, learned it today~